includes/Core/Authentication/Authentication.php

```diff
@@ -1056,7 +1056,7 @@ private function get_reconnect_after_url_mismatch_notice() {
 					$current_url   = $this->context->get_canonical_home_url();
 					$content       = sprintf(
 						'<p>%s <a href="%s">%s</a></p>',
-						esc_html__( 'Looks like the URL of your site has changed. In order to continue using Site Kit, you’ll need to reconnect, so that your plugin settings are updated with the new URL.', 'google-site-kit' ),
+						esc_html__( 'Site Kit by Google: Looks like the URL of your site has changed. In order to continue using Site Kit, you’ll need to reconnect, so that your plugin settings are updated with the new URL.', 'google-site-kit' ),
```

Comment:
While this accomplishes the desired end result, it also creates a new string to be translated unnecessarily.

Instead, we should separate the prefix to be translated separately and then translate the concatenation. Joining translated strings is generally against i18n best practices but in this case the prefix and message are clearly separate, so as long as the concatenation is also translatable, it should be fine.

This way, all the existing translatable strings remain as before, and since the plugin name is already translated, we only need the additional string tying everything together.

e.g.
```php
/* translators: 1: Plugin name. 2: Notice content. */
__( '%1$s: %2$s', 'google-site-kit' )
```

The translated plugin name can be retrieved with
```php
get_plugin_data( GOOGLESITEKIT_PLUGIN_MAIN_FILE )['Name']
```

Reply:
Thank you very much for the kind and explanatory review @aaemnnosttv!

I apologise for not considering this scenario earlier and it completely makes sense. I have addressed this in #5240.

Reply:
😄 no worries, thanks @nfmohit !